libs/gretty/src/main/groovy/org/akhikhl/gretty/AppStartTask.groovy

```diff
@@ -25,6 +27,144 @@ class AppStartTask extends StartBaseTask {
   @Delegate
   private WebAppConfig webAppConfig = new WebAppConfig()
 
+  @Input
+  @Optional
+  getContextPath() {
+    webAppConfig.contextPath
+  }
+
+  @Input
+  @Optional
+  getInitParameters() {
+    webAppConfig.initParameters
+  }
+
+  @Input
+  @Optional
+  getRealm() {
+    webAppConfig.realm
+  }
+
+  @Input
+  @Optional
+  getRealmConfigFile() {
+    webAppConfig.realmConfigFile
+  }
+
+  @Input
+  @Optional
+  getContextConfigFile() {
+    webAppConfig.contextConfigFile
+  }
+
+  @Input
+  @Optional
+  getScanDirs() {
```

Comment:
Is this really an input? I would keep this `@Internal` or only use the path. Since using only the path is more involved (add a separate getter returning the absolute path), I would opt for ignoring this as an input for now.

Reply:
OK will do.